assets/app/view/game/player.rb

```diff
@@ -124,12 +126,17 @@ def render_info
             h(:td, 'Value'),
             h('td.right', @game.format_currency(@game.player_value(@player))),
           ]),
-          h(:tr, [
-            h(:td, 'Liquidity'),
-            h('td.right', @game.format_currency(@game.liquidity(@player))),
-          ]),
         ])
 
+        if @game.show_liquidity?(@player)
```

Comment:
why is this added?

Reply:
The ability to hide the display of liquidity for certain players. For this game union bank doesn't need that shown. It's a value that's not actionable since the bank can never sell shares.

Reply:
seems like there's no need to add this then since there's no harm in showing liquidity for union bank, let's delete it?